artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -3919,7 +3919,10 @@ private void createResources(boolean isAutoCreate, SimpleString destinationAddre
             SimpleString destinationQueueName = prefix.concat(getAddress()).concat(suffix);
             SimpleString filter = new SimpleString(String.format("%s = '%s'", Message.HDR_ORIGINAL_ADDRESS, getAddress()));
             try {
-               server.createQueue(new QueueConfiguration(destinationQueueName).setAddress(destinationAddress).setFilterString(filter).setAutoCreated(true).setAutoCreateAddress(true), true);
+               Queue destinationQueue = server.createQueue(new QueueConfiguration(destinationQueueName).setAddress(destinationAddress).setFilterString(filter).setAutoCreated(true).setAutoCreateAddress(true), true);
+               if (destinationQueue != null && destinationQueue.isSwept()) {
+                  destinationQueue.setSwept(false);
```

Comment:
shouldn't this be done here instead?

https://github.com/apache/activemq-artemis/blob/5ce70f9e37ba1bb7c56372c2e79f9e9d8c9d1fa5/artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java#L4093

Reply:
Meaning.. if anyone called createQueue, and it already existed.. the swept should be set to false I think. that means the fix would apply to any other case where createQueue was called.

Reply:
Sure, that's no problem. The intent being to also catch some similar circumstances outside of the auto created resources, right?

Reply:
The sweep is only used for the reaper.  So it wouldn't hurt to set it to false in there. 

I could say set it if it was auto created.  But it shouldn't matter.

Reply:
I run my complete CI with your changes (before my requested change) and they were all good.

I will merge as soon as you amend the change.

Reply:
Should be done now, hope it looks good... only thing I'm considering is if the `setSwept` should be done before the `ignoreIfExists`-check instead of after? Decided against it for now but I can change again if that makes sense.